app/controllers/conference_registrations_controller.rb

```diff
@@ -33,8 +33,10 @@ def new
   end
 
   def show
-    @total_price = Ticket.total_price(@conference, current_user)
+    TicketPurchase.destroy_all(user_id: current_user.id, conference_id: @conference.id, paid: 'f')
```

Comment:
What's strange is that you are modifying data in the `show` method.
Could you do this somewhere else?